examples/farmer/farmer_cylinders.py

```diff
@@ -77,6 +89,7 @@ def main():
         hub_dict = vanilla.ph_hub(*beans,
                                   scenario_creator_kwargs=scenario_creator_kwargs,
                                   ph_extensions=None,
+                                  PH_converger=PH_converger,
```

Comment:
Perhaps for a different PR: I think `PH_converger` should be renamed `ph_converger` to keep the options consistent.

Reply:
Good point. I should do it on this PR